### PR TITLE
Remove semistandard, extend standard with semi

### PR DESCRIPTION
**Pull request by jacogr**

### CODE REVIEW

File: packages/dev-react/config/tslint.js [hunk 1]
```diff
@@ -3,5 +3,9 @@
 // of the Apache-2.0 license. See the LICENSE file for details.
 
 module.exports = {
-  'extends': '@polkadot/dev/config/tslint'
+  extends: '@polkadot/dev/config/tslint',
+  rulesDirectory: './node_modules/tslint-react/rules/',
```
Comment (amaurymartiny) [inline packages/dev-react/config/tslint.js]: Can it extend an array, and remove this line?

Comment (jacogr) [reply packages/dev-react/config/tslint.js]: Then it adds all of them from `tslint-react`. So rather that overriding/disabling all, just trying to pull in the single rule

Comment (jacogr) [reply packages/dev-react/config/tslint.js]: ... and couldn't find another way to pull in a single rule, `extends` pulls in all

Comment (amaurymartiny) [reply packages/dev-react/config/tslint.js]: Ah yeah true

**Pull request merged**